html/browser/common.css

```diff
@@ -319,4 +319,10 @@ div.notice
 {
 	width: 100%;
 	clear: both;
+}
+
+.charPreview{
+	-ms-interpolation-mode: nearest-neighbor; 
```

Comment:
>and add the other versions of this css tag for the web client
I actually did that the first time, but image-rendering was making DS freeze the first time I opened the character setup.